streaming/base/converters/jsonToMDS.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
1. We should make it clear in the (1) filename, (2) method name, and (3) docstrings that this goes through spark, not directly as a user would probably assume
2. Is there some powerful reason by this is in camel case? Method/file should be named in snake case